pipes/WDL/tasks/tasks_demux.wdl

```diff
@@ -246,7 +249,7 @@ task illumina_demux {
 
   runtime {
     docker: "${docker}"
-    memory: "200 GB"
+    memory: select_first([machine_mem_gb, 240]) + " GB"
```

Comment:
On GCE, an `n1-highmem-32` is 208GB of RAM, which motivated this original value (and I think we never saw the high water mark above 160-180GB) -- setting it to 240 will probably cause it to go for an `n1-standard-64` or something on those platforms. Can we keep it to 200 to fit a better common denominator between EC2 and GCE?

Reply:
Sure thing; I changed it back. The increase was in response to demux failure for a NovaSeq S4 on Terra, but with parameterization it'll be straightforward to re-launch with more memory.